newIDE/app/src/ResourcesList/index.js

```diff
@@ -142,6 +184,8 @@ export default class ResourcesList extends React.Component<Props, State> {
       .getAllResourceNames()
       .toJSArray()
       .map(resourceName => resourcesManager.getResource(resourceName));
+    // console.log(allResourcesList)
+    // allResourcesList.map(resource => console.log(resource.getName()))
```

Comment:
You can remove these commented lines if it's working :)

Reply:
Also don't forget to run prettier on your file ;) (Cmd+Shift+P > Format Document in VSCode with Prettier extension installed)

Reply:
Hehe of course :)
Btw I started using prettier on Yarn too, but it backfired because some of the Yarn devs don't like how prettier formats the code. Had to undo the changes manually xD

Reply:
Yeah using or not prettier is a choice that has to be made on a whole codebase.
It's not an easy decision for a project, so I understand that Yarn devs prefer not to use it.

I still think it's an awesome tool as it allow us as developers not to worry at all about code styling. You just write your code, press a button and boom! It's formatted! It's so fast and I don't have to check the code style at all - just to make sure that you ran prettier :) This is saving a lot of time.

Reply:
I have to say that I quite like it too - after you showed it to me have been using it almost always :)